codewof/static/js/question_types/base.js

```diff
@@ -47,7 +47,7 @@ function update_gamification(data) {
         $("#badge_toast_header").text("New badges!");
         $("#badge_toast_body").text(badges);
```

Comment:
Each badge earned should be on a new line
Consider removing the `- ` prefix for each badge to be consistent with points earned